Remove kernel versions not supported by Armadeus boards from user choice.

Index: buildroot/target/linux/Config.in.advanced
===================================================================
--- buildroot.orig/target/linux/Config.in.advanced	2010-06-17 16:45:13.000000000 +0200
+++ buildroot/target/linux/Config.in.advanced	2010-06-17 16:46:36.000000000 +0200
@@ -98,21 +98,26 @@
 
 config BR2_LINUX_2_6_STABLE
 	bool "The latest stable Linux kernel (2.6.33.4)"
+	depends on !BR2_TARGET_ARMADEUS
 
 config BR2_LINUX_2_6_33
 	bool "Linux 2.6.33"
+	depends on !BR2_TARGET_ARMADEUS
 	select BR2_KERNEL_BASE
 
 config BR2_LINUX_2_6_32
 	bool "Linux 2.6.32"
+	depends on !BR2_TARGET_ARMADEUS
 	select BR2_KERNEL_BASE
 
 config BR2_LINUX_2_6_31
 	bool "Linux 2.6.31"
+	depends on !BR2_TARGET_ARMADEUS
 	select BR2_KERNEL_BASE
 
 config BR2_LINUX_2_6_30
 	bool "Linux 2.6.30"
+	depends on !BR2_TARGET_ARMADEUS
 	select BR2_KERNEL_BASE
 
 config BR2_LINUX_2_6_29
@@ -121,6 +126,7 @@
 
 config BR2_LINUX_2_6_28
 	bool "Linux 2.6.28"
+	depends on !BR2_TARGET_ARMADEUS
 	select BR2_KERNEL_BASE
 
 config BR2_LINUX_2_6_27
